test/serdes_test.c

```diff
@@ -0,0 +1,454 @@
+/*
```

Comment:
Does this file need quite a few
```
#ifndef OPENSSL_NO_RSA
```

Reply:
No.  RSA is not disablable.

The `OPENSSL_NO_RSA` that you see sprinkled in the code are remnants from before 1.1.0, and should really be cleaned away.

Reply:
Ok .. I think I have added some new ones in 3.0 then.

Reply:
(pre-1.1.0 OpenSSL was much wilder...  you could try to disabled *anything*, whether it made sense or not.  Also, I'm sure you remember the time when using the RSA algorithm was legally dodgy, so people had much more reason to disable it at the time than now)